packages/vite/src/node/build.ts

```diff
@@ -232,6 +232,12 @@ export interface BuildOptions {
    * @default true
    */
   reportCompressedSize?: boolean
+  /**
+   * Set to false to disable reporting compressed chunk sizes.
+   * Can slightly improve build speed.
```

Comment:
Since it is disabled by default, it would be better to change both sentences:
```ts
/**
  * Set to true to enable reporting compressed chunk sizes.
  * Enabling this option can slightly deteriorate the build speed.
  * @default false
  */
```

Reply:
if it is enabled by default, change the default.